docs/application/native/guides/ui/efl/wearable/component-circle-slider.md

```diff
@@ -88,16 +90,18 @@ You can modify the circle slider properties using the following functions.
 | `eext_circle_object_color_set()`<br>`eext_circle_object_color_get()` | Set and get the color of circle line and font in the circle object. |
 | `eext_circle_object_disabled_set()`<br>`eext_circle_object_disabled_get()` | Set and get the disabled state of the circle object. |
 
-The circle slider consists of 2 parts, introduced in the following table.
+The circle slider consists of 4 parts, introduced in the following table.
 
 **Table: Circle slider items**
 <table>
 <thead>
 <tr><th>Sample</th><th>Items</th></tr>
 </thead>
 <tbody>
-<tr><td rowspan="2"> <img alt="elm/slider/horizontal/warning" src="./media/circle_slider3.png" /></td>
+<tr><td rowspan="4"> <img alt="elm/slider/horizontal/warning" src="./media/circle_slider2.png" /></td>
 <td> <code>default</code>: Default circle item, which draws a slider bar.<br>This item represents the value of the circle slider, which is a red circular area in the left example.</td></tr>
+<tr><td> <code>cue</code>: Cue circle item, which is a touchable handle located at the end of the slider bar.</td></tr>
+<tr><td> <code>effect</code>: Cue effect circle item, which is a effect render when a cue item is touched.</td></tr>
```

Comment:
Please rephrase this sentence to:

Cue effect circle item, which renders an effect when a cue item is touched.
(or)
Cue effect circle item, which is an effect renderer when a cue item is touched.